### PR TITLE
Auxiliar tsp functions

### PR DESCRIPTION
**Pull request by FranciscoCardoso913**

### CODE REVIEW

File: cmake-build-debug/.ninja_deps
Comment (ZeAntonioM) [inline cmake-build-debug/.ninja_deps:1]: cat > .gitignore
cmake-build-debug

**Review by ZeAntonioM — CHANGES_REQUESTED** [review]
Just change what i said. Apart from that, seems good

File: cmake-build-debug/.ninja_deps
Comment (FranciscoCardoso913) [reply cmake-build-debug/.ninja_deps:1]: I think is supposed to go so that the teacher does not need to compile

**Review by DiogoAlexandreOliveiraDaSilva — APPROVED** [review]
Seems Good

**Pull request merged**